dhiway/Dhiway-Challenge09.md

```diff
@@ -264,6 +261,7 @@ While the code is running, you can watch the events populating on the web consol
     }
     ```
     
+Note that these queries doesn't provide any PII data about seller, store, product or buyer. One has to map the application data through hashing logic to figure out the actual content. This is only to provide public view of activities around the system.
```

Comment:
@smohan-dw @sankarshanmukhopadhyay see if this makes sense. I wanted to add this so people are not confused by not having feilds like 'name', 'model', 'description' etc in the queries. Feel free to suggest better wording here.